cdk/lib/__snapshots__/prism.test.ts.snap

```diff
@@ -1091,95 +1049,6 @@ exports[`The PrismEc2App stack matches the snapshot 1`] = `
       },
       "Type": "AWS::CloudWatch::Alarm",
     },
-    "WazuhSecurityGroup": {
```

Comment:
See https://github.com/guardian/cdk/releases/tag/v61.5.0. Also note as Prism uses the new CFN based ASG deployment, we don't need to stage this change across multiple deployments.